ui/changes/src/graph.js

```diff
@@ -21,80 +21,36 @@ function generateData(count, yrange) {
   return series;
 }
 
-function rerender(connections, teamComponentMapping) {
+async function generateData() {
+  let map = await getComponentRegressionMap();
+  let return_map = [];
+  Object.keys(map)
+    .slice(0, 10)
```

Comment:
I know this is just for testing, but instead of limiting the number of components, you could hardcode a source component/team.

Reply:
I don't see any teams within the Component Regression Map. Is this something I should bring in from outside of that function?

Reply:
Let's hardcode a component for now then, I'll see how to expose the team information (maybe a separate artifact similar to the current one but at the team level rather than the component level).